Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -364,6 +364,7 @@ namespace Multiplayer
     void NetBindComponent::NotifyLocalChanges()
     {
         m_localNotificationRecord.ResetConsumedBits(); // Make sure our consumed bits are reset so that we can run through the notifications
+        m_localNotificationRecord.SetRemoteNetworkRole(m_netEntityRole); // Make sure our record's role is set so the correct set of signals fire
```

Comment:
Doesn’t this incorrectly clobber the remote role sent down from the server during entity replication with our own local role?

Reply:
It's actually clobbering Invalid right now, let me check against the remote role. That sounds more correct though testing was producing reasonable values here.

Reply:
I'd had an assumption here that I verified as acccurate. m_netEntityRole is set based on the entity init from the server and is only manipulated if Controllers are constructed/destructed locally. The choice here was to set m_localNotificationRecord's role at the same time as m_netEntityRole or to ensure it stays in sync. I figured staying in sync makes more sense if for some reason Controllers are constructed/destructed.

Reply:
As mentioned from Joe, this *should* be a no-op, as this is for eventing locally changed network properties. Therefore if this change does make a difference there is something more fundamentally wrong that we should locate and correct.

Reply:
I think this might actually be obsoleted by the change to fix subtracting autonomous dirty bits from records. Removing this.